 2019-09-30 03:20:07,607 [P764/D2/T148] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 03:20:07
 2019-09-30 03:20:15,830 [P764/D2/T156] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 07:05:34,449 [P764/D2/T160] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.85.111:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.85.111:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-30 07:10:55,590 [P764/D2/T160] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.85.111:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.85.111:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-30 07:16:16,652 [P764/D2/T160] ERROR Umbraco.Web.Scheduling.KeepAlive - Failed (at "http://83.41.85.111:8008/umbraco").
System.Net.Http.HttpRequestException: Error al enviar la solicitud. ---> System.Net.WebException: No es posible conectar con el servidor remoto ---> System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder 83.41.85.111:8008
   en System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   en System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   en System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   en System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   en System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   en Umbraco.Web.Scheduling.KeepAlive.<PerformRunAsync>d__5.MoveNext()
 2019-09-30 07:20:07,606 [P764/D2/T79] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 07:20:07
 2019-09-30 07:20:52,296 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Terminating
 2019-09-30 07:20:52,296 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Waiting for tasks to complete
 2019-09-30 07:20:52,309 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Terminating
 2019-09-30 07:20:52,309 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Waiting for tasks to complete
 2019-09-30 07:20:52,309 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Terminating
 2019-09-30 07:20:52,309 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Waiting for tasks to complete
 2019-09-30 07:20:52,309 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Terminating
 2019-09-30 07:20:52,309 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Waiting for tasks to complete
 2019-09-30 07:20:52,333 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Terminating
 2019-09-30 07:20:52,333 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Waiting for tasks to complete
 2019-09-30 07:20:52,333 [P764/D2/T94] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [Umbraco.Web.Scheduling.IBackgroundTask] Tasks completed, terminated
 2019-09-30 07:20:52,333 [P764/D2/T85] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledPublishing] Tasks completed, terminated
 2019-09-30 07:20:52,340 [P764/D2/T15] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [KeepAlive] Tasks completed, terminated
 2019-09-30 07:20:52,340 [P764/D2/T95] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [InstructionProcessing] Tasks completed, terminated
 2019-09-30 07:20:52,333 [P764/D2/T81] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [ScheduledTasks] Tasks completed, terminated
 2019-09-30 07:20:52,340 [P764/D2/T89] INFO  Umbraco.Core.MainDom - Stopping...
 2019-09-30 07:20:52,340 [P764/D2/T89] INFO  Umbraco.Web.ExamineStartup - Examine shutting down
 2019-09-30 07:20:52,387 [P764/D2/T89] INFO  Umbraco.Web.ExamineStartup - Complete (took 30ms)
 2019-09-30 07:20:52,387 [P764/D2/T89] INFO  Umbraco.Core.MainDom - Released MainDom.
 2019-09-30 07:20:52,387 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Terminating
 2019-09-30 07:20:52,387 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Waiting for tasks to complete
 2019-09-30 07:20:52,387 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [HealthCheckNotifier] Tasks completed, terminated
 2019-09-30 07:20:52,434 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Terminating
 2019-09-30 07:20:52,434 [P764/D2/T89] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Waiting for tasks to complete
 2019-09-30 07:20:52,434 [P764/D2/T67] INFO  Umbraco.Web.Scheduling.BackgroundTaskRunner - [LogScrubber] Tasks completed, terminated
 2019-09-30 07:20:52,559 [P764/D2/T90] INFO  Umbraco.Core.UmbracoApplicationBase - Application shutdown. Details: HostingEnvironment

_shutDownMessage=HostingEnvironment initiated shutdown
HostingEnvironment originó el cierre

_shutDownStack=   en System.Environment.GetStackTrace(Exception e, Boolean needFileInfo)
   en System.Environment.get_StackTrace()
   en System.Web.Hosting.HostingEnvironment.InitiateShutdownInternal()
   en System.Web.Hosting.HostingEnvironment.InitiateShutdownWithoutDemand()
   en System.Web.Hosting.PipelineRuntime.StopProcessing()
 2019-09-30 07:30:11,295 [P3320/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco 7.15.3 application starting on SYNOVMW10-UMBRA
 2019-09-30 07:30:11,373 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Determining hash of code files on disk
 2019-09-30 07:30:11,402 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Hash determined (took 43ms)
 2019-09-30 07:30:11,606 [P3320/D2/T1] INFO  Umbraco.Core.MainDom - Acquiring MainDom...
 2019-09-30 07:30:11,606 [P3320/D2/T1] INFO  Umbraco.Core.MainDom - Acquired MainDom.
 2019-09-30 07:30:11,621 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:30:11,812 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 187ms)
 2019-09-30 07:30:11,812 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IApplicationStartupHandler
 2019-09-30 07:30:11,812 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IApplicationStartupHandler (took 3ms)
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2ms)
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyEditorValueConverter (took 1ms)
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-09-30 07:30:11,939 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IPropertyValueConverter
 2019-09-30 07:30:11,949 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IPropertyValueConverter (took 5ms)
 2019-09-30 07:30:11,965 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:30:11,965 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-09-30 07:30:11,965 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.Mvc.SurfaceController
 2019-09-30 07:30:11,965 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.Mvc.SurfaceController (took 1ms)
 2019-09-30 07:30:11,981 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:30:11,981 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 0ms)
 2019-09-30 07:30:11,981 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Web.WebApi.UmbracoApiController
 2019-09-30 07:30:11,981 [P3320/D2/T1] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Web.WebApi.UmbracoApiController (took 3ms)
 2019-09-30 07:30:20,949 [P3320/D2/T1] INFO  Umbraco.Core.DatabaseContext - CanConnect = True
 2019-09-30 07:30:21,825 [P3320/D2/T1] INFO  Umbraco.Web.Cache.CacheRefresherEventHandler - Initializing Umbraco internal event handlers for cache refreshing
 2019-09-30 07:30:22,224 [P3320/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Initializing Examine and binding to business logic events
 2019-09-30 07:30:22,224 [P3320/D2/T1] INFO  Umbraco.Web.Search.ExamineEvents - Adding examine event handlers for index providers: 3
 2019-09-30 07:30:22,451 [P3320/D2/T1] INFO  Umbraco.Core.CoreBootManager - Umbraco application startup complete (took 11444ms)
 2019-09-30 07:30:23,248 [P3320/D2/T12] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - New ApplicationUrl detected: http://umbracotest.caprichosdelucia.com:8008/umbraco
 2019-09-30 07:30:23,248 [P3320/D2/T12] INFO  Umbraco.Core.Sync.ApplicationUrlHelper - ApplicationUrl: http://umbracotest.caprichosdelucia.com:8008/umbraco (UmbracoModule request)
 2019-09-30 07:30:24,856 [P3320/D2/T8] INFO  umbraco.content - Load Xml from file...
 2019-09-30 07:30:24,856 [P3320/D2/T8] INFO  umbraco.content - Loaded Xml from file.
 2019-09-30 07:30:28,341 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:30:28,341 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 3ms)
 2019-09-30 07:30:28,341 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving Umbraco.Core.PropertyEditors.IParameterEditor
 2019-09-30 07:30:28,364 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved Umbraco.Core.PropertyEditors.IParameterEditor (took 3ms)
 2019-09-30 07:30:30,217 [P3320/D2/T8] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 07:30:30,217 [P3320/D2/T12] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 07:31:24,967 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving umbraco.interfaces.IDiscoverable
 2019-09-30 07:31:24,967 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved umbraco.interfaces.IDiscoverable (took 2ms)
 2019-09-30 07:31:24,967 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolving [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper
 2019-09-30 07:31:24,967 [P3320/D2/T8] INFO  Umbraco.Core.PluginManager - Resolved [Umbraco.Core.Persistence.Mappers.MapperForAttribute]Umbraco.Core.Persistence.Mappers.BaseMapper (took 3ms)
 2019-09-30 07:31:25,012 [P3320/D2/T12] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 07:31:24
 2019-09-30 08:31:27,106 [P3320/D2/T150] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:29,107 [P3320/D2/T161] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:32,372 [P3320/D2/T140] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:32,731 [P3320/D2/T159] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:52,387 [P3320/D2/T126] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:54,309 [P3320/D2/T126] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:57,606 [P3320/D2/T159] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 08:31:57,926 [P3320/D2/T158] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 11:31:25,044 [P3320/D2/T142] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 11:31:25
 2019-09-30 12:47:59,560 [P3320/D2/T126] ERROR Umbraco.Web.Strategies.ServerRegistrationEventHandler - Failed to update server record in database.
System.Data.SqlClient.SqlException (0x80131904): Se agotó el tiempo de espera de ejecución. El período de tiempo de espera transcurrió antes de la finalización de la operación o el servidor no responde. ---> System.ComponentModel.Win32Exception (0x80004005): Tiempo de espera de la operación de espera agotado
   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   en System.Data.SqlClient.TdsParser.Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj)
   en System.Data.SqlClient.TdsParser.TdsExecuteTransactionManagerRequest(Byte[] buffer, TransactionManagerRequestType request, String transactionName, TransactionManagerIsolationLevel isoLevel, Int32 timeout, SqlInternalTransaction transaction, TdsParserStateObject stateObj, Boolean isDelegateControlRequest)
   en System.Data.SqlClient.SqlInternalConnectionTds.ExecuteTransactionYukon(TransactionRequest transactionRequest, String transactionName, IsolationLevel iso, SqlInternalTransaction internalTransaction, Boolean isDelegateControlRequest)
   en System.Data.SqlClient.SqlInternalTransaction.Commit()
   en System.Data.SqlClient.SqlTransaction.Commit()
   en Umbraco.Core.Persistence.Database.CleanupTransaction()
   en Umbraco.Core.Persistence.Database.CompleteTransaction()
   en Umbraco.Core.Scoping.Scope.DisposeLastScope()
   en Umbraco.Core.Scoping.Scope.Dispose()
   en Umbraco.Core.Persistence.UnitOfWork.ScopeUnitOfWork.DisposeResources()
   en Umbraco.Core.DisposableObjectSlim.Dispose(Boolean disposing)
   en Umbraco.Core.DisposableObjectSlim.Dispose()
   en Umbraco.Core.ObjectExtensions.DisposeIfDisposable(Object input)
   en Umbraco.Core.DisposableObjectSlim.Dispose(Boolean disposing)
   en Umbraco.Core.DisposableObjectSlim.Dispose()
   en Umbraco.Core.Persistence.LockingRepository`1.WithWriteLocked(Action`1 action, Boolean autoCommit)
   en Umbraco.Core.Services.ServerRegistrationService.TouchServer(String serverAddress, String serverIdentity, TimeSpan staleTimeout)
   en Umbraco.Web.Strategies.ServerRegistrationEventHandler.TouchServerTask.PerformRun()
ClientConnectionId:d8b8d080-8b6a-4aec-bb99-fee2f37b2d30
Error Number:-2,State:0,Class:11
 2019-09-30 12:48:34,168 [P3320/D2/T152] ERROR Umbraco.Core.Persistence.UmbracoDatabase - Exception (c1f052ca).
System.InvalidOperationException: Operación no válida, se ha terminado la conexión.
   en System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()
   en System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()
   en System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()
   en System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)
   en System.Data.SqlClient.TdsParserStateObject.TryPeekByte(Byte& value)
   en System.Data.SqlClient.SqlDataReader.TrySetMetaData(_SqlMetaDataSet metaData, Boolean moreInfo)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   en System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   en System.Data.SqlClient.SqlDataReader.get_MetaData()
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en Umbraco.Core.Persistence.PetaPocoCommandExtensions.<>c__DisplayClass5_0.<ExecuteReaderWithRetry>b__0()
   en Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func)
   en Umbraco.Core.Persistence.Database.<Query>d__74`1.MoveNext()
 2019-09-30 12:48:34,168 [P3320/D2/T152] ERROR Umbraco.Web.BatchedDatabaseServerMessenger+InstructionProcessing - Failed (will repeat).
System.InvalidOperationException: SqlTransaction se completó; ya no se puede utilizar.
   en System.Data.SqlClient.SqlTransaction.ZombieCheck()
   en System.Data.SqlClient.SqlTransaction.Rollback()
   en Umbraco.Core.Persistence.Database.CleanupTransaction()
   en Umbraco.Core.Persistence.Database.AbortTransaction()
   en Umbraco.Core.Scoping.Scope.DisposeLastScope()
   en Umbraco.Core.Scoping.Scope.Dispose()
   en Umbraco.Web.BatchedDatabaseServerMessenger.InstructionProcessing.TryPerformRun()
   en Umbraco.Web.BatchedDatabaseServerMessenger.InstructionProcessing.PerformRun()
 2019-09-30 12:55:27,855 [P3320/D2/T140] WARN  Umbraco.Web.Mvc.RenderRouteHandler - The current Document Type home matches a locally declared controller of type UmbracoTesting.Controllers.HomeController. Custom Controllers for Umbraco routing must implement 'Umbraco.Web.Mvc.IRenderController' and inherit from 'System.Web.Mvc.ControllerBase'.
 2019-09-30 15:31:25,059 [P3320/D2/T75] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 15:31:25
 2019-09-30 19:31:25,059 [P3320/D2/T162] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 19:31:25
 2019-09-30 21:18:38,030 [P3320/D2/T139] ERROR Umbraco.Web.BatchedDatabaseServerMessenger+InstructionProcessing - Failed (will repeat).
System.Data.SqlClient.SqlException (0x80131904): Se ha establecido la conexión con el servidor correctamente, pero se ha producido un error durante el inicio de sesión previo del protocolo de enlace. (provider: SSL Provider, error: 0 - Tiempo de espera de la operación de espera agotado.) ---> System.ComponentModel.Win32Exception (0x80004005): Tiempo de espera de la operación de espera agotado
   en System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)
   en System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   en System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   en System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   en System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   en System.Data.SqlClient.SqlConnection.Open()
   en Umbraco.Core.Persistence.FaultHandling.RetryPolicy.<>c__DisplayClass20_0.<ExecuteAction>b__0()
   en Umbraco.Core.Persistence.FaultHandling.RetryPolicy.ExecuteAction[TResult](Func`1 func)
   en Umbraco.Core.Persistence.Database.OpenSharedConnection()
   en Umbraco.Core.Persistence.Database.BeginTransaction(IsolationLevel isolationLevel)
   en Umbraco.Core.Scoping.Scope.get_Database()
   en Umbraco.Core.Sync.DatabaseServerMessenger.ProcessDatabaseInstructions()
   en Umbraco.Core.Sync.DatabaseServerMessenger.Sync()
   en Umbraco.Web.BatchedDatabaseServerMessenger.InstructionProcessing.TryPerformRun()
   en Umbraco.Web.BatchedDatabaseServerMessenger.InstructionProcessing.PerformRun()
ClientConnectionId:be4407ea-033c-4909-afcd-599100b7b7c8
Error Number:258,State:0,Class:20
 2019-09-30 23:31:25,076 [P3320/D2/T130] INFO  umbraco.BusinessLogic.Log - Log scrubbed.  Removed all items older than 2019-09-29 23:31:25
